src/StaticWebAssetsSdk/Tasks/ResolveCompressedAssets.cs

```diff
@@ -0,0 +1,266 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.Extensions.FileSystemGlobbing;
+
+namespace Microsoft.AspNetCore.StaticWebAssets.Tasks;
+
+public class ResolveCompressedAssets : Task
```

Comment:
Log pretty much everything as shown below so that we can easily troubleshoot binlogs
https://github.com/dotnet/sdk/blob/main/src/StaticWebAssetsSdk/Tasks/DefineStaticWebAssets.cs#L100